src/Infrastructure/LeanCode.ForceUpdate/LeanCode.ForceUpdate.Services/VersionHandler.cs

```diff
@@ -0,0 +1,34 @@
+using Microsoft.AspNetCore.Http;
+
+namespace LeanCode.ForceUpdate.Services;
+
+public class VersionHandler
+{
+    public virtual Task<VersionSupportResult> CheckVersionAsync(
```

Comment:
I'll be very nitty here, but I don't like how this work with `VersionSupportQH` :upside_down_face: 

I guess this is virtual so that projects may change the logic here freely. I think the interface exposed is rigid though, because it expects you to pass minimum required versions.

What if you have just `CheckVersionAsync(Version version, Platform platform, HttpContext context)` and get minimal versions as dependencies?